### PR TITLE
DG Update for my features

### PR DESCRIPTION
**Pull request by SpecOps2016**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -112,11 +112,78 @@ Examples:
...
-### 1.5.3 Sorting bills
+Format:
+```sortpatient <prefix><input> ...```
+
+* The command word is `sortpatient`.
+* The prefixes are c/ for Criteria and o/ for Order.
+* The criteria can be Name of patient (name), Phone number of patient (phone), Email address of patient (email), Address of patient (address).
+* The order can be Ascending (asc) or Descending (desc).
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
```
Comment (guokweijie) [inline docs/UserGuide.md:127]: yours doesn't need this

File: docs/UserGuide.md [hunk 2]
```diff
@@ -112,11 +112,78 @@ Examples:
...
+
+Format:
+```sortappointment <prefix><input> ...```
+
+* The command word is `sortappointment`.
+* The prefixes are c/ for Criteria and o/ for Order.
+* The criteria can be Name of patient (name), Medical Test of appointment (test), Slot of appointment (slot) and Doctor of appointment (doctor).
+* The order can be Ascending (asc) or Descending (desc).
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
```
Comment (guokweijie) [inline docs/UserGuide.md:150]: again don't need this

File: docs/UserGuide.md [hunk 3]
```diff
@@ -112,11 +112,78 @@ Examples:
...
+### 1.5.3 Sorting bills `sortbill`
+
+Sorts bills by a single field
+
+Format:
+```sortbill <prefix><input> ...```
+
+* The command word is `sortbill`.
+* The prefixes are c/ for Criteria and o/ for Order.
+* The criteria can be Name of patient (name), Amount (amount), Date (date), Payment status (status).
```
Comment (guokweijie) [inline docs/UserGuide.md:170]: bill date not date

File: docs/UserGuide.md [hunk 4]
```diff
@@ -112,11 +112,78 @@ Examples:
...
+* The order can be Ascending (asc) or Descending (desc).
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
```
Comment (guokweijie) [inline docs/UserGuide.md:173]: unnecessary

File: docs/UserGuide.md [hunk 5]
```diff
@@ -112,11 +112,78 @@ Examples:
...
+* If the criteria is Payment status, Ascending will show bills which are paid first and Descending will show bills which are unpaid first.
+* If there are no prefixes keyed in, an error message will be shown with the correct command format.
+* If the input after a prefix is empty/invalid, an error message with the constraint of the field will be shown.
```
Comment (guokweijie) [inline docs/UserGuide.md:176]: repeated

File: docs/UserGuide.md [hunk 6]
```diff
@@ -181,37 +278,42 @@ After:
...
+|              | an appointment of a patient          | `findappointment`   | `fa`     | `n/` Name of patient<br/>`s/` Slot of appointment<br/>`d/` Doctor name of the appointment<br/>`t/` Medical test of patient                                                                                                                                                                                                                                                                                              |
+|              | a bill to an appointment             | `findbill`          | `fb`     | `n/` Name of patient<br/>`a/` Amount of bill<br/>`d/` Bill date of bill                                                                                                                                                                                                                                                                                                                                                 |
```
Comment (guokweijie) [inline docs/UserGuide.md:298]: payment status also

**Review by guokweijie — DISMISSED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (SpecOps2016) [reply docs/UserGuide.md:403]: talked about this. More information is good

File: docs/UserGuide.md (hunk 3 above)
Comment (SpecOps2016) [reply docs/UserGuide.md:423]: Minor change. Fixed

File: docs/UserGuide.md (hunk 5 above)
Comment (SpecOps2016) [reply docs/UserGuide.md]: fixed

File: docs/UserGuide.md (hunk 6 above)
Comment (SpecOps2016) [reply docs/UserGuide.md]: fixed

**Review by TZL0 — APPROVED** [review]
LGTM

**Pull request merged**